test/Microsoft.Azure.SignalR.Tests.Common/ParameterDelegator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR.Tests.Common
+{
+    public class ParameterDelegator
```

Comment:
can we use `ConfigurationBuilder().AddInMemoryCollection()` like https://msdn.microsoft.com/en-us/magazine/mt814420.aspx?f=255&MSPPError=-2147217396 instead of `ParameterDelegator`?